### PR TITLE
Pb 3

### PR DESCRIPTION
**Pull request by kuriakopoulosgiorgos**
Minor fixes in the models, also added the end point to list all meetup events with the url : {domain}/api/events

### CODE REVIEW

**Review by dimgrav — APPROVED** [review]
Kudos for the Saturday night work! :+1:

File: events/api/urls.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from events.api import views
+
+urlpatterns = [
+    path('', views.EventsListAPIView().as_view(), name='events'),
```
Comment (dimgrav) [inline events/api/urls.py:6]: will have to re-check that tomorrow, before merging

File: prizy/urls.py [hunk 2]
```diff
@@ -30,7 +30,7 @@
     path('verify/', verify_jwt_token, name='jwt_verify'),
     # API routes
     path('api/', include('accounts.urls')),
-    path('api/', include('events.urls'))
+    path('api/events/', include('events.api.urls'), name='events-api'),
```
Comment (dimgrav) [inline prizy/urls.py:33]: Do we really need this change?

File: events/api/serializers.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+            'event_fee',
+            'description',
+            'event_url',
+            'yes_reservation_count',
+            'duration',
+            'name',
+            'time',
+            'updated',
+            'status',
+
```
Comment (dimgrav) [inline events/api/serializers.py:55]: Make sure to remove unnecessary whitespace!

File: events/api/urls.py (hunk 1 above)
Comment (dimgrav) [reply events/api/urls.py]: Better to provide the root events path explicitly:

`path('events', views.EventsListAPIView().as_view(), name='events'),`

File: prizy/urls.py (hunk 2 above)
Comment (dimgrav) [reply prizy/urls.py:33]: OK, I checked your repo and it makes sense now, but why overcomplicate things? We could move everything to the events root directory!

Comment (kuriakopoulosgiorgos) [reply prizy/urls.py:33]: If this is making confusion I am going to fix that soon, is a convention I follow when it comes for APIs

Comment (dimgrav) [reply prizy/urls.py:33]: No, it's a great convention actually, but we don't have that many endpoints to handle, so it might be a bit "over-engineered" in this case. And since we didn't have that structure in `accounts`, I wanted to be consistent throughout the codebase.

We can definitely reorganize the api structure, but after discussion, to make sure we're all on the same page!

Check [PB-3 on Trello](https://trello.com/c/nLdkCnjt/2-pb-3-add-endpoint-to-list-meetup-events-to-registered-users), I've added some comments there as well!
